.bazelrc

```diff
@@ -0,0 +1,22 @@
+build --cxxopt=-Wno-format
+build --cxxopt=-std=c++14
+build --cxxopt=-fdiagnostics-color=always
```

Comment:
this is for having errors/warning output colored when compiling. might be not needed in bazel